src/Http/Exception/TooManyRequestsException.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Http\Exception;
+
+use Exception;
+
+/**
+ * Class TooManyRequestsException.
+ *
+ * @author Arman Tuyakbayev <arman.tuyakbayev@rebilly.com>
+ * @version 0.1
+ */
+final class TooManyRequestsException extends ClientException
+{
+    private $retryAfter = '';
+    private $rateLimit = 0;
+    private $rateRemaining = 0;
+
+    public function __construct($message = "", $retryAfter = '', $rateLimit = 0, $rateRemaining = 0, $code = 0, Exception $previous = null)
```

Comment:
I propose to use specific arguments first, and the parent constructor arguments in the end of list.